bin/stashcp

```diff
@@ -275,6 +276,14 @@ def parse_job_ad():
 
 def dostashcpdirectory(sourceDir, destination, cache, debug=False):
     sourceItems = subprocess.Popen(["xrdfs", stash_origin, "ls", sourceDir], stdout=subprocess.PIPE).communicate()[0].split()
+    
+    # If the cache is not specified by the command line, then look for the closest
+    # Do it here, so that we only do it once per stashcp call.  Though, if everything is in
+    # CVMFS, then this is unncessary.
```

Comment:
I don't like that this is done here because as the comment says, if everything is in cvmfs, it is unnecessary.  You could check for cvmfs first, and make sure that all subsequent copies only use cvmfs.  Or you could use a static or global variable for the cache name, and only set it if it is None.